ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -165,9 +165,9 @@ def any(self, axis=None, dtype=None, out=None, keepdims=False, *, where=True):
     def argsort(self, *, axis=-1, kind=None, order=None):
         return np_frontend.argsort(self, axis=axis, kind=kind, order=order)
 
-    def mean(self, *, axis=None, dtype=None, out=None, keepdims=False, where=True):
+    def mean(self, axis=None, dtype=None, out=None, keepdims=False, *, where=True):
         return np_frontend.mean(
-            self,
+            self.ivy_array,
```

Comment:
I don't think `.ivy_array` is needed here. `self` is a frontend array and frontend functions are designed to handle this type of input.

Reply:
yes right, will update it